### ISSUE TITLE
Error installing into parent path /Users/tony/src/vcr/CHANGELOG.md

### ISSUE THREAD

**Issue by superplussed** [vcr/vcr#582]
gem install vcr                                                                            
`
Fetching: vcr-3.0.2.gem (100%)
ERROR:  While executing gem ... (Gem::Package::PathError)
    installing into parent path /Users/tony/src/vcr/CHANGELOG.md of /Users/jeremysmith/.rvm/gems/ruby-2.3.1/gems/vcr-3.0.2 is not allowed
`

**Comment by mcfiredrill** [comment]
I had the same issue, apparentally it is fixed in rubygems 2.5.2.
https://github.com/rubygems/rubygems/issues/1448

**Comment by mcfiredrill** [comment]
`gem update --system` fixed it for me.

**Comment by shlima** [comment]
+1 

```bash
Gem::Package::PathError: installing into parent path /Users/tony/src/vcr/CHANGELOG.md of /Users/alex/.rvm/gems/ruby-2.3.1/gems/vcr-3.0.2 is not allowed

```

**Comment by mcfiredrill** [comment]
@shaicoleman Did you try updating rubygems?

**Comment by morgoth** [comment]
Have the same issue on travis.
You can fix it by adding to `.travis.yml`
```
before_install:
  - gem update --system
```

**Comment by superplussed** [comment]
@mcfiredrill do you know how it happened in the first place?  I brought it up to our team this morning, and we spent like 30 minutes trying to figure out how a local reference of /Users/tony got into rubygems, we couldn't figure it out!  It's not in the vcr repo, it's not in the gemspec, so where is the reference?

**Comment by mcfiredrill** [comment]
@superplussed That's the path to the source code on my machine. I'm really sorry about this.

I don't quite understand the details yet, I'm pretty the sure problem is upstream in rubygems, in the aforementioned issue. I simply built the gem and pushed it in the normal fashion.

**Comment by jasnow** [comment]
I have the issue too.

**Comment by mcfiredrill** [comment]
I will try building a new gem with version 2.2 and pushing it, but until then you can fix this by updating your rubygems:
```
gem update --system
```

**Comment by superplussed** [comment]
No problem @mcfiredrill, we were more curious than anything.  Jut a strange occurrence especially when you can't track down why it happened.  I appreciate all of the work you put into this!  👍

**Comment by jasnow** [comment]
Edge version of vcr gem fixed the issue for me:
 *   gem 'vcr', github: 'vcr/vcr'

**Comment by TheMetalCode** [comment]
Just ran into this myself and I was like "who the heck is Tony and why is this gem trying to install stuff into his folder?" :)

**Comment by asaaki** [comment]
What happens to all the Tonys?

**Comment by mcfiredrill** [comment]
Just pushed 3.0.3, built with ruby 2.2. Hopefully this Gem doesn't have the same issues.

**Comment by jasnow** [comment]
Verified - Fixed all of the repos I used **vcr* in. THANKS.

**Comment by mcfiredrill** [comment]
Going to close as this seems to be fixed with 3.0.3. Feel free to comment if you are still having issues.
